lib/query/query.server.js

```diff
@@ -19,11 +19,13 @@ export default class Query extends Base {
     }
 
     /**
-     * @param args
+     * @param context
      * @returns {*}
      */
-    fetchOne(...args) {
-        return _.first(this.fetch(...args));
+    fetchOne(context = {}) {
+        context.$options = context.$options || {};
+        context.$options.limit = 1;
+        return this.fetch(context);
```

Comment:
@Floriferous 

Wouldn't this return the array here? Meaning we still need _.first()?

Reply:
Indeed, fixed now